source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,63 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', async () => {
```

Comment:
Fails `yarn lint`: `warning  Promise returned in function argument where a void return was expected  @typescript-eslint/no-misused-promises`